src/components/StealthCopiable/index.tsx

```diff
@@ -38,20 +38,24 @@ type StealthCopiableProps = {
   side?: 'left' | 'right'
   copyText?: string
   copiedText?: string
+  hide?: boolean
 }
 
 const StealthCopiable: FunctionComponent<StealthCopiableProps> = ({
   children,
   side = 'right',
   copyText = 'Copy',
   copiedText = 'Copied',
+  hide = false,
 }) => {
   const string = recursivelyGetChildrenString(children)
 
   const [isCopied, setCopied] = useClipboard(string, {
     successDuration: 5000,
   })
 
+  if (hide) return <StyledContainer>{children}</StyledContainer>
```

Comment:
Can't this be handled by the parent ? This is not really a Copiable component with hide=true :)

Reply:
@chambo-e You're right, component should stay pure. I will make a custom component that will handle this case directly in the project